tests/schemas/input/test_json.py

```diff
@@ -83,6 +91,15 @@ def test_basic_data_schema_validates(csl_data_validator):
     csl_data_validator.validate(csl)
 
 
+def test_basic_citation_schema_validates(csl_citation_validator):
+    cite = [{
```

Comment:
Either my python code is wrong, or my json fragment here is.

Reply:
The error from CI:

```
E           jsonschema.exceptions.ValidationError: [{'id': 'example-id', 'type': 'report', 'title': 'test'}] is not of type 'object'
```

I think it's because the top-level should be a python dictionary not a list according to:

https://github.com/citation-style-language/schema/blob/7b3108e6eca2e4dd48c154cf650a8da64c1932f0/schemas/input/csl-citation.json#L5

Try removing `[` and `]`.